### ISSUE TITLE
invalid linker name in argument '-fuse-ld=lld'

### ISSUE THREAD

**Issue by alizahedi94** [ish-app/ish#1264]
Hi

I have installed LLVM with "brew install llvm" and when I try to build for iOS in Xcode I still get the error "invalid linker name in argument '-fuse-ld=lld'"

How can I get this project to run?

Thanks

**Comment by emkey1** [comment]
You might try my custom version of iSH, specifically...

https://github.com/emkey1/ish-AOK/tree/golden6

I'm fairly sure I got that to build on my M1 MacBook Air.

Ignore the golden7 version, it has some issues.

If you try it and it doesn't work, please leave me an issue on my fork and I'll see if I can get you going.

**Comment by saagarjha** [comment]
@alizahedi94 Are you building on Apple silicon?

@emkey1 we should probably just fix this upstream first, to be honest.

**Comment by alizahedi94** [comment]
@emkey1 Since your project is a fork how do I download the submodules?

@saagarjha Yes

**Comment by emkey1** [comment]
> @emkey1 Since your project is a fork how do I download the submodules?
> 
> @saagarjha Yes

Clone the whole thing and do 

git switch golden6

To be sure you're in the right branch.

**Comment by emkey1** [comment]
> @alizahedi94 Are you building on Apple silicon?
> 
> @emkey1 we should probably just fix this upstream first, to be honest.

Agreed, but I'm honestly not sure what fixed it.  There were two things I recall.  One was adding the brew bin directory to the path and the other was using the full path to the non Apple clang for one of the modules.

Comparing the new might shed some light but there have been a lot of changes in my fork and I'm not all that great at keeping things clean git wise.

**Comment by alizahedi94** [comment]
@emkey1 

Tried your project and still getting /opt/homebrew/bin/clang:1:1: invalid linker name in argument '-fuse-ld=lld'

**Comment by vareille** [comment]
apk add lld

**Comment by 0x7c7** [comment]
this is almost 3 years old

**Comment by Bernix01** [comment]
`brew install lld` also works...
